packages/create-react-native-library/templates/native-library-new/react-native.config.js

```diff
@@ -5,7 +5,11 @@ module.exports = {
   dependency: {
     platforms: {
       android: {
+        <%_ if (local) { -%>
+        cmakeListsPath: 'build/generated/source/codegen/jni/CMakeLists.txt',
+        <%_ } else { -%>
         cmakeListsPath: 'generated/jni/CMakeLists.txt',
+        <%_ } -%>
```

Comment:
```suggestion
        <% if (example === 'vanilla') { -%>
        cmakeListsPath: 'generated/jni/CMakeLists.txt',
        <% } else { -%>
        cmakeListsPath: 'build/generated/source/codegen/jni/CMakeLists.txt',
        <% } -%>
```
Let's swap the order so we also cover the test app.